pages.it/common/xkill.md

```diff
@@ -0,0 +1,8 @@
+# xkill
+> Termina una finestra interattivamente attraverso la sessione grafica.
+> Vedi anche `kill` e `killall`
+
+- Mostra un cursore per terminate una finestra cliccandoci sopra con il tasto
+  sinistro del mouse (premendo qualsiasi altro tasto si cancella l'azione):
```

Comment:
Though this line is long, let's not break it, because I don't think clients support that.
```suggestion
- Mostra un cursore per terminate una finestra cliccandoci sopra con il tasto sinistro del mouse (premendo qualsiasi altro tasto si cancella l'azione):
```

Reply:
True, these linebreaks were automatically placed by my text editor and I didn't notice them.